specification/components/examples/ResponseRequestForDocumentsAccess.yaml

```diff
@@ -1,9 +1,10 @@
 summary: |
-  Request for documents access for historical permissions that is pending.
+  A pending response from a request where the patient asked for detailed access
```

Comment:
Maybe could be reworded "A response from a request where the patient asks for detailed access to the historical medical record. Response indicates the request is pending review"